state_machine/plugs/flight/lib/pycubed.py

```diff
@@ -41,34 +41,125 @@
 
 # NVM register numbers
 # TODO: confirm registers start in MRAM partition & update board build file
-_FLAG = const(20)
-_DWNLINK = const(4)
-_DCOUNT = const(3)
-_RSTERRS = const(2)
-_BOOTCNT = const(0)
-_GSRSP = const(10)
+_FLAG     = const(20)
+_DWNLINK  = const(4)
+_DCOUNT   = const(3)
+_RSTERRS  = const(2)
+_BOOTCNT  = const(0)
 
 class Satellite:
     # Define NVM flags
-    f_deploy   = bitFlag(register=_FLAG, bit=1)
-    f_mdeploy  = bitFlag(register=_FLAG, bit=2)
-    f_burn1    = bitFlag(register=_FLAG, bit=3)
-    f_burn2    = bitFlag(register=_FLAG, bit=4)
+    f_deploy = bitFlag(register=_FLAG, bit=1)
+    f_mdeploy = bitFlag(register=_FLAG, bit=2)
+    f_burn1 = bitFlag(register=_FLAG, bit=3)
+    f_burn2 = bitFlag(register=_FLAG, bit=4)
 
     # Define NVM counters
-    c_boot       = multiBitFlag(register=_BOOTCNT, lowest_bit=0, num_bits=8)
-    c_state_err  = multiBitFlag(register=_RSTERRS, lowest_bit=4, num_bits=4)
-    c_vbus_rst   = multiBitFlag(register=_RSTERRS, lowest_bit=0, num_bits=4)
-    c_deploy     = multiBitFlag(register=_DCOUNT,  lowest_bit=0, num_bits=8)
-    c_downlink   = multiBitFlag(register=_DWNLINK, lowest_bit=0, num_bits=8)
-    c_gs_resp    = multiBitFlag(register=_GSRSP,   lowest_bit=0, num_bits=8)
+    c_boot = multiBitFlag(register=_BOOTCNT, lowest_bit=0, num_bits=8)
+    c_state_err = multiBitFlag(register=_RSTERRS, lowest_bit=4, num_bits=4)
+    c_vbus_rst = multiBitFlag(register=_RSTERRS, lowest_bit=0, num_bits=4)
+    c_deploy = multiBitFlag(register=_DCOUNT, lowest_bit=0, num_bits=8)
+    c_downlink = multiBitFlag(register=_DWNLINK, lowest_bit=0, num_bits=8)
 
+    # change to 433?
     UHF_FREQ = 433.0
+    
+    def __init_sun_sensors(self, i2c1, i2c2):
```

Comment:
I'd say let's put the `__init_xxx` functions after the `__init__` function.

Reply:
Also, since `self` has access to `i2c1` and `i2c2`, why do we need to pass them into this function separately?